.tests/test-workflow/workflow/envs/snappy.post-deploy.sh

```diff
@@ -0,0 +1,15 @@
+#!env bash
+set -o pipefail
+conda init
+conda activate "$CONDA_PREFIX"
+
+if [[ -z "${GITHUB_WORKSPACE}" ]]; then
```

Comment:
I think you way want to test for **non-empty** variable here (`-n` instead of `-z`)

Reply:
Whoops, indeed.